.github/workflows/python-app.yml

```diff
@@ -0,0 +1,106 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
```

Comment:
These comments don't seem relevant?

Reply:
Oops, this is left over from the template and I forgot to remove it